wiki/BanchoBot/es.md

```diff
@@ -1,155 +1,154 @@
+Ejemplos de enviar este comando se muestran abajo:
 
 ```
-pippi: !roll 1000
-BanchoBot: pippi rolls 109 point(s)
+13:00 pippi: !roll 1000
+13:00 BanchoBot: pippi rolls 109 point(s)
 ```
 
-### STATS
-
 ```
-!STATS <usuario>
+13:01 pippi: !roll probability of failure
+13:01 BanchoBot: pippi rolls 75 point(s)
 ```
 
-Muestra las estadísticas para el usuario introducido. Los resultados despenderán del modo que esté jugando o del último que haya jugado.
-
-*Nota: BanchoBot no te dirá de qué modo de juego ha sacado esas estadísticas.*
-
-*Aviso: BanchoBot responderá con un `User not found` si el usuario nunca ha jugado partidas, aunque el usuario exista.*
-
-#### Ejemplos de uso
+### Stats
 
 ```
-pippi: !stats peppy
-BanchoBot: Stats for peppy:
-BanchoBot: Store: 422,187,979 (#94718)
-BanchoBot: Plays: 7149 (lv65)
-BanchoBot: Accuracy: 87.29%
+!stats <usuario>
 ```
 
+`!stats` muestra las estadísticas y el estado del usuario introducido. El resultado dependerá del [modo de juego](/wiki/Game_mode) que haya jugado, aunque BanchoBot no mostrará que modo de juego provienen. Si se pide que muestre las estadísticas de un usuario que nunca ha jugado osu!, BanchoBot respondera con `User not found` (usuario no encontrado), sin importar de que el usuario sí exista. Ejemplo de enviar este comando se muestra abajo:
+
 ```
-pippi: !stats Ephemeral:
-BanchoBot: Stats for Ephemeral is Playing:
-BanchoBot: Store: 2,384,155,536 (#33697)
-BanchoBot: Plays: 14054 (lv96)
-BanchoBot: Accuracy: 94.19%
+13:01 pippi: !stats peppy
+13:01 BanchoBot: Stats for peppy:
+13:01 BanchoBot: Score: 427,514,691 (#94718)
+13:01 BanchoBot: Plays: 7348 (lv66)
+13:01 BanchoBot: Accuracy: 87.13%
 ```
 
-### WHERE
+En total son 7 estados que se pueden mostrar por el comando `!stats`: Editing, Idle, Lobby, Modding, Multiplayer, Multiplaying y Playing. Ejemplo de enviar este comando de un usuario con un estado disponible se muestra abajo:
 
 ```
-!WHERE <usuario>
+13:01 pippi: !stats peppy
+13:01 BanchoBot: Stats for peppy is Playing:
+13:01 BanchoBot: Score: 427,514,691 (#94718)
+13:01 BanchoBot: Plays: 7348 (lv66)
+13:01 BanchoBot: Accuracy: 87.13%
 ```
 
-Muestra la ubicación actual del usuario
+### Where
 
-*Nota: Por predeterminado, esto solo muesta el país del usuario. Si el usuario tiene `Compartir tu ciudad actual con otros` habilitado, tambien puede mostrar su ciudad.*
+```
+!where <usuario>
+```
 
-#### Ejemplos de uso
+`!where` muestra la ubicación actual del usuario. Por defecto, esto solo muestra el país del usuario. Si el usuario tiene la opción `Compartir tu ciudad actual con otros` habilitado, también puede mostrar su ciudad. Ejemplo de enviar este comando se muestra abajo:
 
 ```
-pippi: !where Ephemeral
-BanchoBot: Ephemeral is in Australia
+13:02 pippi: !where Ephemeral
+13:02 BanchoBot: Ephemeral is in Australia
 ```
 
 ### FAQ
 
 ```
-!FAQ <entrada>
-!FAQ list
+!faq <entrada>
+!faq list
 ```
```

Comment:
@TicClick Can you edit this? I can't create a review
![image](https://user-images.githubusercontent.com/58957468/195165984-dbabb06c-4747-44e2-8136-0a6477fa1542.png)